backend/molgenis-emx2-sql/src/main/java/org/molgenis/emx2/sql/SqlSchemaMetadataExecutor.java

```diff
@@ -71,6 +71,12 @@ static void executeCreateSchema(SqlDatabase db, SchemaMetadata schema) {
         .execute("GRANT USAGE ON SCHEMA {0} TO {1}", name(schema.getName()), name(aggregator));
     db.getJooq().execute("GRANT ALL ON SCHEMA {0} TO {1}", name(schema.getName()), name(manager));
 
+    // add collation
+    db.getJooq()
+        .execute(
+            "CREATE COLLATION IF NOT EXISTS {0}.numeric (provider = icu, locale = 'en-u-kn-true');",
```

Comment:
could this be a named entity within 'MOLGENIS' schema only and then you refer to that one all the time? Or is there added value to have one per schema? Then you don't need this